src/runtime/polyfill/performance.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  performance,
+  Performance,
+  PerformanceEntry,
+  PerformanceMark,
+  PerformanceMeasure,
+  PerformanceObserver,
+  PerformanceObserverEntryList,
+  PerformanceResourceTiming,
+} from "../node/perf_hooks/index";
```

Comment:
Most of these APIs are Web APIs supported by Node.js too. For a generic polyfill/shim, I would suggest that we start implementing them in `src/web/*` ([Performance](https://developer.mozilla.org/en-US/docs/Web/API/Performance)). 

If there are slight differences or more Node.js-specific exports we can override them via `node/perf_hooks` but even if identical, a clear export would keep it more future proof because other than Node.js, there would be JS runtimes that need Web standard impl of performances API.

Reply:
ok I took a swing at restructuring, lmk if this is what you had in mind.

I think something is a little off though now with the `web` entrypoint, because although this seems to build fine it's failing in my entrypoint with `[ERROR] Could not resolve "../../web/performance"`. It could be an issue with the way that our test harness is using esbuild.

Reply:
Ah ok perfect! 

Yeah I think this will probably solve the issues 👍